packages/core/src/sass/app/_common/layout/footer.scss

```diff
@@ -149,3 +150,35 @@
         }
     }
 }
+
+/** @define footer-regulatory-information */
+.footer-regulatory-information {
+    padding: 2rem;
+
+    p {
+        @include typeface(--paragraph-left-normal-prominent);
+    }
+    &__link {
+        @include typeface(--paragraph-left-bold-prominent);
+        color: var(--brand-red-coral);
+        text-decoration: none;
+    }
+    &__icon {
+        display: flex;
+        justify-content: center;
+        align-items: flex-start;
+        margin-bottom: 2.4rem;
+    }
+}
+
+/** @define footer-icon-separator */
+.footer-icon-separator {
+    width: 1px;
+    height: 1.6rem !important;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: var(--general-section-1);
+    content: ' ';
```

Comment:
I don't think you need `content` here since it's not a `pseudo` element.

Reply:
Oh yes, I was going down the path of adding pseudo element, ended up not doing that. this is a leftover. thanks for the feedback.